sys/aarch64/boot.c

```diff
@@ -17,9 +18,23 @@
     __rv;                                                                      \
   })
 
+#define BOOT_KASAN_SANITIZED_SIZE(end)                                         \
+  roundup2(roundup2((vaddr_t)(end), PAGESIZE) - KASAN_SANITIZED_START,         \
+           SUPERPAGESIZE * KASAN_SHADOW_SCALE_SIZE)
+
 /* Last physical address used by kernel for boot memory allocation. */
 __boot_data void *_bootmem_end;
 
+static __noreturn __used void aarch64_boot(paddr_t dtb, paddr_t pde);
+
+/*
+ * Temporary stack in VA.
+ * It's needed because in rpi3.c accesses to stack are instrumented
```

Comment:
This comment is obsolete. `rpi3.c` (?), GCC -> the compiler. Note that this comment will become obsolete when we stop mapping the boot segment.

Reply:
Removed!